src/wallet/wallet.cpp

```diff
@@ -2540,14 +2540,16 @@ CAmount CWallet::GetLegacyBalance(const isminefilter& filter, int minDepth, cons
     return balance;
 }
 
-void CWallet::AvailableCoins(std::vector<COutput>& vCoins, bool fOnlySafe, const CCoinControl *coinControl, bool fIncludeZeroValue, AvailableCoinsType nCoinType, bool fUseInstantSend) const
+void CWallet::AvailableCoins(std::vector<COutput> &vCoins, bool fOnlySafe, const CCoinControl *coinControl, const CAmount &nMinimumAmount, const CAmount &nMaximumAmount, const CAmount &nMinimumSumAmount, const uint64_t &nMaximumCount, const int &nMinDepth, const int &nMaxDepth, AvailableCoinsType nCoinType, bool fUseInstantSend) const
```

Comment:
There are many `AvailableCoins` calls left with invalid parameters now. Please check calls in:
1. SelectPSInOutPairsByDenominations
2. SelectPrivateCoins
3. GetCollateralTxDSIn
4. GetMasternodeOutpointAndKeys
5. HasCollateralInputs
6. CreateTransaction
7. masternode_outputs

Reply:
resolved